x-pack/solutions/security/packages/kbn-securitysolution-exception-list-components/src/list_header/menu_items/index.tsx

```diff
@@ -96,6 +96,7 @@ const MenuItemsComponent: FC<MenuItemsProps> = ({
             onClick={() => {
               if (typeof onManageRules === 'function') onManageRules();
             }}
+            disabled={isReadonly}
```

Comment:
The usual best practice is to explain why the button is disabled. It's not hard to imagine popping up SDHs as users may think the button is disabled due to a bug.